src/fileformat/types/resource_table/resource.cpp

```diff
@@ -336,6 +336,17 @@ void Resource::setSublanguageId(std::size_t rId)
 	sublanguageIdIsValid = true;
 }
 
+/**
+ * @brief Checks if the offset is valid, with UINT32_MAX reserved as invalid
+ * 
+ * @return true 
+ * @return false 
+ */
+bool Resource::isValidOffset() const
+{
+	return this->offset != UINT32_MAX ? true : false;
```

Comment:
Is this `this` really necessary here? Also condition `<cond> ? true : false` can be simplified just as `<cond>`.

Reply:
Thanks, fixed!